### DISCUSSION TITLE
pc端搞不明白，只能手机端看看了

### DISCUSSION THREAD

**Discussion by LHk416** [share121/inter-knot#4617] [绝区零]
![IMG_20240908_154452](https://github.com/user-attachments/assets/b1bae0bb-44b3-46a5-807e-8dfa22ec8f43)

**Comment by xingyuehao** [comment]
pc端需要下载油猴用来启动绳网插件，然后需要科学上网注册github账户才能用（像米游社表情包需要额外下载插件一起用）<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by canyusuo** [reply]
米游社的表情包要下啥插件，有链接吗我看原作者没发耶

**Reply by Dave-12138** [reply]
> 米游社的表情包要下啥插件，有链接吗我看原作者没发耶

表情包是 #4639 所述的用户脚本，网页端绳网没做反向代理所以只好装[这个浏览器插件](https://microsoftedge.microsoft.com/addons/detail/allow-cors-accesscontro/bhjepjpgngghppolkjdhckmnfphffdag?hl=zh-CN&gl=CN)凑合一下

**Reply by canyusuo** [reply]
> > 米游社的表情包要下啥插件，有链接吗我看原作者没发耶
> 
> 表情包是 #4639 所述的用户脚本，网页端绳网没做反向代理所以只好装[这个浏览器插件](https://microsoftedge.microsoft.com/addons/detail/allow-cors-accesscontro/bhjepjpgngghppolkjdhckmnfphffdag?hl=zh-CN&gl=CN)凑合一下

感谢，也是用上了<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/b171cc5b6c8d8b6257a3bf8163fb2680_567796524796359289.png" alt="阿姬得意" width="75" >

**Comment by For-elysia** [comment]
老八好看，偷了
